src/Overlay.js

```diff
@@ -19,18 +19,14 @@ class OverlayRaw extends React.Component {
     if (Object.getOwnPropertyNames(this.props.Combatant).length === 0)
       return false
 
-    let maxRows = this.props.config.maxCombatants
-    let dataArray = Object.keys(this.props.Combatant)
-    let battler = dataArray.slice(0, maxRows)
-    let combatant
+    const maxRows = this.props.config.maxCombatants //not sure why let was being used here
```

Comment:
Probably because I changed it after, I was learning es6 at this time and didn't have a linter on - good catch though :)